R/adjust-mh.R

```diff
@@ -0,0 +1,173 @@
+#' @importFrom stats weighted.mean
+estimate.mh <- function(df.mh, estimand) {
+  with(df.mh, {
+    delta <- y1.k - y0.k
+    if (estimand == "MH") {
+      weight <- n.1k * n.0k / (n.1k + n.0k)
+      delta_all <- weighted.mean(delta, weight)
+    } else{
+      n = sum(n.1k + n.0k)
```

Comment:
Curious if you have a reason for switching from `<-` to `=`? Would be good to make them consistent, I would prefer `<-`. Applies throughout the pull request.